packages/astro/package.json

```diff
@@ -17,6 +17,12 @@
     "node": ">=18.14.1"
   },
   "type": "module",
+  "files": [
+    "cjs",
```

Comment:
m (not just relevant for this file but for all):

Right now, the `files` array's contents point to paths relative to `<package>/build`. This conflicts a bit with the entry points semantics where we point to files from `<package>`. There's good reason for both but we could consider unifying this to `<package>` to avoid confusion.

WDYT? I don't have a strong opinion that we should do this because I'm well aware of this fact but maybe it'd be worth doing so. The downside is that we need to rewrite the array in `prepack.ts` just like we do for the entry points.

If our plan is still to remove prepacking, feel free to completely disregard this.